### PR TITLE
hide password input

### PR DESCRIPTION
**Pull request by DanielRuf**

### CODE REVIEW

File: src/Services/IoService.php [hunk 1]
```diff
@@ -122,10 +122,15 @@ public function writeln($message)
      * @param  null            $default
      * @return string
      */
-    public function ask($question, $default = null)
+    public function ask($question, $default = null, $hidden = false)
     {
         $question = $question instanceof Question ? $question : new Question($question, $default);
 
+        if($hidden){
```
Comment (FiveDigital) [inline src/Services/IoService.php]: A space after the `if` is missing.

Comment (DanielRuf) [reply src/Services/IoService.php]: Thanks for the hint, I'll push a new commit this evening.

**Comment by DanielRuf** [comment]
I've pushe a new commit =)

**Comment by mitelg** [comment]
your PR looks good to me :+1: 
thanks for your contribution @DanielRuf

**Comment by DanielRuf** [comment]
You are welcome.
Thanks for merging.

**Pull request merged**